JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -77,14 +81,24 @@ fun JetnewsApp(
                 // Only enable opening the drawer via gestures if we allow showing it
                 gesturesEnabled = allowDrawerToBeShown
```

Comment:
This isn't new with this PR, but I noticed that the drawer doesn't take insets into account:

![device-2021-09-17-141424](https://user-images.githubusercontent.com/4217560/133854479-c3f364d1-05b1-461f-9fcc-1194780f1cdb.png)

This is a Pixel 3 XL emulator, and the drawer doesn't take the notch into account:

<img width="399" alt="Screen Shot 2021-09-17 at 2 15 40 PM" src="https://user-images.githubusercontent.com/4217560/133854445-0292e82b-b010-4e3c-ad85-2d57b8d850b1.png">
s

Reply:
Fixed as well!